packages/elements/src/datetime-picker/index.ts

```diff
@@ -942,6 +944,14 @@ export class DatetimePicker extends ControlElement implements MultiValue {
     const newValues = this.values;
     if (!this.timepicker && newValues[0] && (this.range ? newValues[1] : true)) {
       this.setOpened(false);
+
+      /** There is a known issue when selecting on the custom cell,
```

Comment:
Custom cell selection delegates focus back to the text field when the overlay is closed, causing a sync problem between the calendar and text field. A workaround involves blurring the text field again.